diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowPath.js b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowPath.js
index 91b918d..999ccd8 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowPath.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowPath.js
@@ -28,15 +28,15 @@ var import_getElbowArrowInfo = require("./elbow/getElbowArrowInfo");
 function getArrowBodyPath(shape, info, opts) {
   switch (info.type) {
     case "straight":
-      return new import_PathBuilder.PathBuilder().moveTo(info.start.point.x, info.start.point.y, { offset: 0, roundness: 0 }).lineTo(info.end.point.x, info.end.point.y, { offset: 0, roundness: 0 }).toSvg(opts);
+      return new import_PathBuilder.PathBuilder().moveTo(info.start.point.x, info.start.point.y, { roundness: 0 }).lineTo(info.end.point.x, info.end.point.y, { roundness: 0 }).toSvg(opts);
     case "arc":
-      return new import_PathBuilder.PathBuilder().moveTo(info.start.point.x, info.start.point.y, { offset: 0, roundness: 0 }).circularArcTo(
+      return new import_PathBuilder.PathBuilder().moveTo(info.start.point.x, info.start.point.y, { roundness: 0 }).circularArcTo(
         info.bodyArc.radius,
         !!info.bodyArc.largeArcFlag,
         !!info.bodyArc.sweepFlag,
         info.end.point.x,
         info.end.point.y,
-        { offset: 0, roundness: 0 }
+        { roundness: 0 }
       ).toSvg(opts);
     case "elbow": {
       const path = new import_PathBuilder.PathBuilder();
diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
index 98079f6..cda5c8c 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
@@ -670,8 +670,8 @@ class ArrowShapeUtil extends import_editor.ShapeUtil {
                 style: "draw",
                 randomSeed: shape.id,
                 strokeWidth: 1,
-                passes: 1,
-                offset: 0,
+                passes: 2,
+                offset: strokeWidth * 2,
                 roundness: strokeWidth * 2,
                 props: { strokeWidth: void 0 }
               } : { style: "solid", strokeWidth: 1, props: { strokeWidth: void 0 } }
@@ -868,7 +868,9 @@ const ArrowSvg = (0, import_editor.track)(function ArrowSvg2({
                   style: shape.props.dash,
                   strokeWidth,
                   forceSolid: isForceSolid,
-                  randomSeed: shape.id
+                  randomSeed: shape.id,
+                  passes: 2,
+                  offset: strokeWidth * 2
                 })
               ]
             }
diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/arrowheads.js b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/arrowheads.js
index fb527b1..4dbe893 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/arrowheads.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/arrowheads.js
@@ -29,13 +29,13 @@ function getArrowPoints(info, side, strokeWidth) {
     case "straight": {
       const opposite = side === "end" ? info.start.point : info.end.point;
       const compareLength = import_editor.Vec.Dist(opposite, point);
-      const length = (0, import_editor.clamp)(compareLength / 5, strokeWidth, strokeWidth * 3);
+      const length = (0, import_editor.clamp)(compareLength / 5, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       int = import_editor.Vec.Nudge(point, opposite, length);
       break;
     }
     case "arc": {
       const compareLength = Math.abs(info.bodyArc.length);
-      const length = (0, import_editor.clamp)(compareLength / 5, strokeWidth, strokeWidth * 3);
+      const length = (0, import_editor.clamp)(compareLength / 5, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       const intersections = (0, import_editor.intersectCircleCircle)(
         point,
         length,
@@ -48,7 +48,7 @@ function getArrowPoints(info, side, strokeWidth) {
     case "elbow": {
       const previousPoint = side === "end" ? info.route.points[info.route.points.length - 2] : info.route.points[1];
       const previousSegmentLength = import_editor.Vec.ManhattanDist(previousPoint, point);
-      const length = (0, import_editor.clamp)(previousSegmentLength / 2, strokeWidth, strokeWidth * 3);
+      const length = (0, import_editor.clamp)(previousSegmentLength / 2, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       int = previousPoint ? import_editor.Vec.Nudge(point, previousPoint, length) : point;
       break;
     }
diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/shared/PathBuilder.js b/node_modules/tldraw/dist-cjs/lib/shapes/shared/PathBuilder.js
index 762117a..d282952 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/shared/PathBuilder.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/shared/PathBuilder.js
@@ -413,7 +413,7 @@ class PathBuilder {
     const {
       strokeWidth,
       randomSeed,
-      offset: defaultOffset = strokeWidth / 3,
+      offset: defaultOffset = strokeWidth * 3.5,
       roundness: defaultRoundness = strokeWidth * 2,
       passes = 2,
       onlyFilled = false
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowPath.mjs b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowPath.mjs
index ed9bbaf..db41d65 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowPath.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowPath.mjs
@@ -4,15 +4,15 @@ import { getRouteHandlePath } from "./elbow/getElbowArrowInfo.mjs";
 function getArrowBodyPath(shape, info, opts) {
   switch (info.type) {
     case "straight":
-      return new PathBuilder().moveTo(info.start.point.x, info.start.point.y, { offset: 0, roundness: 0 }).lineTo(info.end.point.x, info.end.point.y, { offset: 0, roundness: 0 }).toSvg(opts);
+      return new PathBuilder().moveTo(info.start.point.x, info.start.point.y, { roundness: 0 }).lineTo(info.end.point.x, info.end.point.y, { roundness: 0 }).toSvg(opts);
     case "arc":
-      return new PathBuilder().moveTo(info.start.point.x, info.start.point.y, { offset: 0, roundness: 0 }).circularArcTo(
+      return new PathBuilder().moveTo(info.start.point.x, info.start.point.y, { roundness: 0 }).circularArcTo(
         info.bodyArc.radius,
         !!info.bodyArc.largeArcFlag,
         !!info.bodyArc.sweepFlag,
         info.end.point.x,
         info.end.point.y,
-        { offset: 0, roundness: 0 }
+        { roundness: 0 }
       ).toSvg(opts);
     case "elbow": {
       const path = new PathBuilder();
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
index 295f3ac..6c0e4a3 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
@@ -686,8 +686,8 @@ class ArrowShapeUtil extends ShapeUtil {
                 style: "draw",
                 randomSeed: shape.id,
                 strokeWidth: 1,
-                passes: 1,
-                offset: 0,
+                passes: 2,
+                offset: strokeWidth * 2,
                 roundness: strokeWidth * 2,
                 props: { strokeWidth: void 0 }
               } : { style: "solid", strokeWidth: 1, props: { strokeWidth: void 0 } }
@@ -884,7 +884,9 @@ const ArrowSvg = track(function ArrowSvg2({
                   style: shape.props.dash,
                   strokeWidth,
                   forceSolid: isForceSolid,
-                  randomSeed: shape.id
+                  randomSeed: shape.id,
+                  passes: 2,
+                  offset: strokeWidth * 2
                 })
               ]
             }
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/arrow/arrowheads.mjs b/node_modules/tldraw/dist-esm/lib/shapes/arrow/arrowheads.mjs
index 0e5a12d..0224e1d 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/arrow/arrowheads.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/arrow/arrowheads.mjs
@@ -13,13 +13,13 @@ function getArrowPoints(info, side, strokeWidth) {
     case "straight": {
       const opposite = side === "end" ? info.start.point : info.end.point;
       const compareLength = Vec.Dist(opposite, point);
-      const length = clamp(compareLength / 5, strokeWidth, strokeWidth * 3);
+      const length = clamp(compareLength / 5, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       int = Vec.Nudge(point, opposite, length);
       break;
     }
     case "arc": {
       const compareLength = Math.abs(info.bodyArc.length);
-      const length = clamp(compareLength / 5, strokeWidth, strokeWidth * 3);
+      const length = clamp(compareLength / 5, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       const intersections = intersectCircleCircle(
         point,
         length,
@@ -32,7 +32,7 @@ function getArrowPoints(info, side, strokeWidth) {
     case "elbow": {
       const previousPoint = side === "end" ? info.route.points[info.route.points.length - 2] : info.route.points[1];
       const previousSegmentLength = Vec.ManhattanDist(previousPoint, point);
-      const length = clamp(previousSegmentLength / 2, strokeWidth, strokeWidth * 3);
+      const length = clamp(previousSegmentLength / 2, Math.max(strokeWidth * 2, 10), Math.max(strokeWidth * 4, 18));
       int = previousPoint ? Vec.Nudge(point, previousPoint, length) : point;
       break;
     }
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/shared/PathBuilder.mjs b/node_modules/tldraw/dist-esm/lib/shapes/shared/PathBuilder.mjs
index 0a352e4..634fa74 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/shared/PathBuilder.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/shared/PathBuilder.mjs
@@ -405,7 +405,7 @@ class PathBuilder {
     const {
       strokeWidth,
       randomSeed,
-      offset: defaultOffset = strokeWidth / 3,
+      offset: defaultOffset = strokeWidth * 3.5,
       roundness: defaultRoundness = strokeWidth * 2,
       passes = 2,
       onlyFilled = false
